client/landing/stepper/declarative-flow/new-hosted-site-flow-user-included.ts

```diff
@@ -4,14 +4,13 @@ import { NEW_HOSTED_SITE_FLOW_USER_INCLUDED } from '@automattic/onboarding';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { addQueryArgs } from '@wordpress/url';
 import { useEffect, useLayoutEffect } from 'react';
-import { useDispatch as reduxUseDispatch } from 'react-redux';
 import { recordFreeHostingTrialStarted } from 'calypso/lib/analytics/ad-tracking/ad-track-trial-start';
 import {
 	setSignupCompleteSlug,
 	persistSignupDestination,
 	setSignupCompleteFlowName,
 } from 'calypso/signup/storageUtils';
-import { useSelector } from 'calypso/state';
+import { useDispatch as reduxUseDispatch, useSelector } from 'calypso/state';
```

Comment:
There were type errors here trying to use dispatch straight from react-redux, as they aren't typed to support thunks.

TODO - verify using calypso/state useDispatch works from these stepper flows and that this doesn't break anything.

Reply:
note it looks like there are some places in stepper already using this version of useDispatch https://github.com/Automattic/wp-calypso/blob/64dc80d4412606544a6f0e32febf52f618c7d251/client/landing/stepper/declarative-flow/site-setup-flow.ts#L11

Reply:
It seems that calypso/state useDispatch is [literally just the same useDispatch from react-redux](https://github.com/Automattic/wp-calypso/blob/64dc80d4412606544a6f0e32febf52f618c7d251/client/state/index.ts#L57), but[ typed to support thunks.](https://github.com/Automattic/wp-calypso/blob/64dc80d4412606544a6f0e32febf52f618c7d251/client/state/types.ts#L32) so this seems safe.